runtime/js/99_main.js

```diff
@@ -1,9 +1,22 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 "use strict";
 
-// Removes the `__proto__` for security reasons.
+// Disables setting `__proto__` and emits a warning instead, for security reasons.
 // https://tc39.es/ecma262/#sec-get-object.prototype.__proto__
-delete Object.prototype.__proto__;
+// deno-lint-ignore prefer-primordials
+Object.defineProperty(Object.prototype, "__proto__", {
```

Comment:
These should use primordials like `ObjectDefineProperty`, `ObjectPrototype` and `ObjectGetPrototypeOf`. You will need to move this after line 53